README.md

```diff
@@ -19,7 +19,21 @@ cd frontend-burgernabije-besluitendatabank
 npm install
 ```
 
-From there, you can use `npm run dev` in order to use the mock API, `npm run prod` in order to use a local BNB api, or `npm run --proxy=https://burgernabije-besluitendatabank-dev.s.redhost.be/` to run using the BNB API on [the external dev server](https://burgernabije-besluitendatabank-dev.s.redhost.be/).
+From there, you can use `npm run dev:proxy`, or any of the following npm scripts:
+
+
+|  npm run...  |  Description |
+| ------------ | ------------ |
+| build        | Creates a production-ready static build |
+| lint         | Runs the linter and returns any errors/warnings. *Is run automatically before committing, cancelling the commit on error* |
+| lint:fix     | Runs the linter, attempting to automatically fix any errors |
+| dev          | Run a development server with the mock api |
+| dev:proxy    | Run a development server with a proxy to [the external dev server](https://burgernabije-besluitendatabank-dev.s.redhost.be/) |
+| start        | Run a development server with the mock api |
+| prod         | Run a production server |
+| test         | Run the linter and then ember tests |
```

Comment:
I'm not sure about this. Just looking at the scripts config in the package.json file gives you the almost same info and it's always up to date.

Reply:
That is fair! I've mainly added it because it might help onboarding, especially since some of the info is a bit more obscured (for example that dev uses a mock API and dev:proxy's url is our external dev server). It also obscures npm run lint:hbs:fix and the other sub-scripts. However, if the maintenance is too big of a possible maintenance hazard, I can understand if it can't make it in!